### PR TITLE
MODDATAIMP-431: Update module readme files to reflect Iris environment variables.

### PR DESCRIPTION
**Pull request by VRohach**
Update README.md.

### CODE REVIEW

**Review by KaterynaSenchenko — COMMENTED** [review]
Please add a note that these variables are relevant for Iris release (and the module version)

**Comment by VRohach** [comment]
> Please add a note that these variables are relevant for Iris release (and the module version)

Added. Could you please verify it?

File: README.md [hunk 1]
```diff
@@ -89,6 +89,31 @@ curl -w '\n' -X POST -D -   \
...
+* "_srm.kafka.RawMarcChunkConsumer.instancesNumber_": 5
+* "_srm.kafka.StoredMarcChunkConsumer.instancesNumber_": 5
+* "_srm.kafka.DataImportConsumersVerticle.instancesNumber_": 5
+* "_srm.kafka.DataImportJournalConsumersVerticle.instancesNumber_": 5
+* "_srm.kafka.RawChunksKafkaHandler.maxDistributionNum_": 100
+* "_srm.kafka.CreatedRecordsKafkaHandler.maxDistributionNum_": 100
+* "_srm.kafka.DataImportConsumer.loadLimit_": 5
+
+####Note
+These variables are relevant for the **Iris** release. Module version: 3.0.0 (3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5).
```
Comment (KaterynaSenchenko) [inline README.md:114]: let's make it bold

Comment (VRohach) [reply README.md]: Done.

**Review by KaterynaSenchenko — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

**Pull request merged**